README.md

```diff
@@ -16,42 +64,72 @@ The [v4_beta_deprecated](https://github.com/aws/aws-iot-device-sdk-embedded-C/tr
 
 All of the released versions of the AWS IoT Embedded C SDK libraries are available as git tags. For example, the last release of the v3 SDK version is available at [tag 3.0.1](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.0.1).
 
+## Versioning
+
+C-SDK releases will now follow a date based versioning scheme with the format YYYYMM.NN_major, where: 
+
+* Y represents the year.
+* M represents the month.
+* N represents the release order within the designated month (00 being the first release).
+* A "major" denotation indicates the addition of new features or significant updates to multiple libraries. 
+
+For example, a second release in June 2021 would be 202106.01. Although the SDK releases have moved to date-based versioning, each library within the SDK will still retain semantic versioning. In semantic versioning, the version number itself (X.Y.Z) indicates whether the release is a major, minor, or point release. You can use the semantic version of a library to assess the scope and impact of a new release on your application. 
+
+## Cloning
+This repo uses [Git Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) to bring in dependent components.
+
+Note: If you download the ZIP file provided by GitHub UI, you will not get the contents of the submodules (The ZIP file is also not a valid git repository).  If you download from the [202009.00 Release Page](https://github.com/aws/aws-iot-device-sdk-embedded-C/releases/tag/202009.00) page, you will get the entire repository (including the submodules) in the ZIP file, aws-iot-device-sdk-embedded-c-202009.00.zip.  
+
+
+To clone using HTTPS:
+```
+git clone https://github.com/aws/aws-iot-device-sdk-embedded-C.git --recurse-submodules
+```
+Using SSH:
+```
+git clone git@github.com:aws/aws-iot-device-sdk-embedded-C.git --recurse-submodules
+```
+
+If you have downloaded the repo without using the `--recurse-submodules` argument, you need to run:
+```
+git submodule update --init --recursive
+```
+
+
 ## Building and Running Demos
 
   
+The libraries in this SDK are not dependent on any operating system. However, the demos for the libraries in this SDK are built and tested on a Linux platform. This SDK builds with [CMake](https://cmake.org/), a cross-platform build tool.
 
-The libraries in this SDK are not dependent on any operating systems. However, the demos for the libraries in this SDK are built and tested on a Linux platform. This SDK builds with [CMake](https://cmake.org/), a cross-platform build tool.
 
+### Prerequisites
 
+- CMake 3.13.0 or later 
 
-### Prerequisites
+- C90 compiler
 
-- CMake 3.13.0 or later and a C90 compiler.
+- Although not a part of the C90 standard, `stdint.h` is required for fixed-width integer types (e.g int32_t).
```

Comment:
Can we point to an alternate stdint.h for non-conforming systems, or at least say which types are needed?

Reply:
At this point, I will choose the latter option. If we provide an alternative header file, we should test with it also.

Reply:
Have provided the list of types that we use from `stdint.h`